tests/e2e/framework/kubernetes.go

```diff
@@ -333,63 +353,70 @@ func (k *KubeInfo) Teardown() error {
 	if *skipSetup || *skipCleanup {
 		return nil
 	}
+	if *installer == "helm" {
+		// clean up using helm
+		err := util.HelmDelete("istio")
+		if err != nil {
+			return nil
+		}
+	} else {
+		if *useAutomaticInjection {
```

Comment:
I think that automatic injection is unrelated to helm install or kubectl install . @ayj  what do you think ?

Reply:
I think we suppose automatic injection for helm, @ayj or @yusuoh ?

Reply:
helm supports automatic injection (see https://github.com/istio/istio/blob/ca3049444afd3fc3170db9d7c7e47533be528eb7/install/kubernetes/helm/istio/values.yaml#L186).